### PR TITLE
Identify the 'Shares' space by its uuid

### PR DESCRIPTION
**Pull request by TheOneRing**
Fixes: #10196

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests-oc10</strong> https://drone.owncloud.com/owncloud/client/14047/6/1

:boom: The GUI tests failed.

GUI Logs: https://cache.owncloud.com/public/owncloud/client/14047/oc10/guiReportUpload/index.html

<details>

Server Logs: https://cache.owncloud.com/public/owncloud/client/14047/oc10/guiReportUpload/serverlog.log

Screenshots:
- [change_collaborator_permissions_of_a_file_&_folder.png](https://cache.owncloud.com/public/owncloud/client/14047/oc10/guiReportUpload/screenshots/change_collaborator_permissions_of_a_file_&_folder.png)

</details>

File: src/libsync/graphapi/drives.cpp [hunk 1]
```diff
@@ -29,8 +29,7 @@ namespace {
 
 const auto mountpointC = QLatin1String("mountpoint");
 const auto personalC = QLatin1String("personal");
-const auto shareC = QLatin1String("virtual");
-
+const auto sharesIdC = QLatin1String("a0ca6a90-a365-4782-871e-d44447bbc668$a0ca6a90-a365-4782-871e-d44447bbc668");
```
Comment (fmoc) [inline src/libsync/graphapi/drives.cpp:32]: We should probably document in code where to find this over at the other repository.

**Review by fmoc — CHANGES_REQUESTED** [review]

File: src/libsync/graphapi/drives.cpp (hunk 1 above)
Comment (TheOneRing) [reply src/libsync/graphapi/drives.cpp:32]: @micbar @C0rby could you point me to the location its defined?

Comment (C0rby) [reply src/libsync/graphapi/drives.cpp:32]: It's defined here: https://github.com/cs3org/reva/blob/edge/pkg/utils/utils.go#L56-L59
And assigned here: https://github.com/cs3org/reva/blob/edge/internal/grpc/services/sharesstorageprovider/sharesstorageprovider.go#L371-L375

Comment (TheOneRing) [reply src/libsync/graphapi/drives.cpp:32]: thx

**Review by fmoc — APPROVED** [review]

**Pull request merged**